bindings/wasm/examples/src/utils_web.js

```diff
@@ -55,12 +41,10 @@ export function logToScreen(message) {
         "<br/>-------------------------------------<br/>";
 }
 
-/**
- *
- * @param {string} url
- */
-export function logExplorerUrlToScreen(url) {
-    logToScreen(`Explorer URL: <a target="_blank" href="${url}"> ${url} </a>`);
+export function linkify(inputText) {
+
+    return inputText.replace(LINK_REGEX, '<a href="$1" target="_blank">$1</a>');
+
```

Comment:
Please comment functions in general, especially when they're somewhat obscure at first glance.